### DISCUSSION TITLE
arm64 support

### DISCUSSION THREAD

**Discussion by wboerhout** [mahmoudbahaa/outlook-for-linux#10] [Ideas]
Are there plans for a release on arm64 ? I am slowly moving away from a Windows 10 laptop and into using a Raspberry Pi 5 for my desktop activities. This project looks promising...

**Comment by gj241** [comment]
Yes — the maintained fork **outlook-for-linux v1.4.0** now ships arm64/aarch64 builds: https://github.com/gj241/outlook-for-linux/releases/tag/v1.4.0

For a Raspberry Pi 5 (which runs a 64-bit aarch64 OS by default), grab the arm64 assets on that release:

- `outlook-for-linux_1.4.0_arm64.deb` (Debian/Ubuntu/RPi OS 64-bit)
- `outlook-for-linux-1.4.0-arm64.AppImage`
- `outlook-for-linux-1.4.0.aarch64.rpm` (Fedora/etc.)
- `outlook-for-linux-1.4.0-arm64.tar.gz`

The upstream repo hadn't been updated in ~3 years, so this is being continued as a maintained fork (gj241/outlook-for-linux). v1.4.0 also adds multi-account support and native GNOME desktop notifications.

One caveat: there's no arm64 **snap** yet (only amd64 and armv7l/armhf snaps) — but the arm64 .deb and AppImage are the better fit for a Pi 5 anyway.
